src/main/java/com/accesso/challengeladder/controller/LoginController.java

```diff
@@ -0,0 +1,34 @@
+package com.accesso.challengeladder.controller;
+
+import com.accesso.challengeladder.requests.LoginRequest;
+import com.accesso.challengeladder.model.User;
+import com.accesso.challengeladder.services.UserService;
+import com.accesso.challengeladder.utils.JsonUtil;
+import com.google.gson.Gson;
+import org.eclipse.jetty.http.HttpStatus;
+import static spark.Spark.post;
+
+import java.io.IOException;
+import java.sql.SQLException;
+
+public class LoginController
+{
+    public LoginController() throws SQLException, IOException
+    {
+        UserService userService = new UserService();
+        post("/login", (req, res) -> {
```

Comment:
What do you want me to change here? should i roll back the changes to create the login endpoint?